### ISSUE TITLE
what do i need to do next

### ISSUE THREAD

**Issue by vanixvanii** [sundowndev/phoneinfoga#1352]
help me please it is showing this error while ive gave go run main.go

web/client.go:14:12: pattern client/dist/*: no matching files found
lib/remote/scanner.go:6:2: package plugin is not in GOROOT (/usr/src/plugin)

will you help me please @sundowndev

**Comment by sundowndev** [comment]
Read [docs](https://sundowndev.github.io/phoneinfoga/contribute/) to build from source

**Comment by vanixvanii** [comment]
yeah i did like that doc said and when i do the npm run build command it starts and shows like this :                      

┌──(root㉿KALI)-[/home/vanix/phoneinfoga/web/client]
└─# npm run build

> client@0.1.0 build
> vue-cli-service build

sh: 1: vue-cli-service: not found

why this is showing that is not found?

**Comment by vanixvanii** [comment]
>

**Comment by pierceparks** [comment]
I am a twitch streamer and I was able to go through my ex-girlfriend’s phone using this hacker ( HACKERSPYTECH @ GMAIL . COM ) I found positive comments about him on my discord group and decided to give it a shot, It still amazes me till now cuz I never thought I could legit be going through her phone from my device without her knowledge. I’m talking front camera and microphone activation, I could view all her photos documents, GPS location and real time screen mirroring. Go through this hacker if you are looking for a reliable hacker to help spy on your cheating partner.
